services/updatefile/updatefile.go

```diff
@@ -0,0 +1,52 @@
+package updatefile
+
+import (
+	"academy/model"
+	"academy/services/dataload"
+	"encoding/csv"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func readCsvFile(filePath string) [][]string {
+	f, err := os.Open(filePath)
+	if err != nil {
+		fmt.Println("Unable to read input file "+filePath, err)
```

Comment:
Reminder for this file to handle your errors and return if something goes wrong

Reply:
Added a bit more error handling